content/en/contribute/code/core/update-dependencies.md

```diff
@@ -51,12 +52,44 @@ Then for each folder go through these steps.
   ```
 
 - Make sure the version of `api/enketo-xslt` is the same as `webapp/enketo-core/enketo-transformer/enketo-xslt`.
-- Don't update `lodash/core` since 4.17.21 is failing, this is an [open issue](https://github.com/lodash/lodash/issues/4904) in their repository.
-- Don't update `helmet` since it is not supporting NodeJS 8 anymore. Its minimum is NodeJS 10+.
-- If you have trouble upgrading any other dependency and you think it'll be challenging to fix it then raise a new issue to upgrade just that dependency. Don't hold up all the other upgrades you've made.
+- Don't update `lodash` to version `4.17.21`, there's a runtime issue when importing `lodash/core` in Admin app:
+  ```
+  UnhandledPromiseRejectionWarning: TypeError: stack.get is not a function
+    at equalObjects (/home/user/app/node_modules/lodash/core.js:1303:28)
+    at baseIsEqualDeep (/home/user/app/node_modules/lodash/core.js:729:18)
+    at baseIsEqual (/home/user/app/node_modules/lodash/core.js:664:12)
+    at Function.isEqual (/home/user/app/node_modules/lodash/core.js:2660:12)
+    at estimateCurrentScheduleDimLevel.then.dimLevel (/home/user/app/src/Devices/SmartLight/NASLightDevice.ts:642:24)
+  ```
+  In their code, the issue is here:
+  ```
+  if (!_.isEqual(oldProfileConfig, newProfileConfig)) {
+    this.setProfileConfig(newProfileConfig);
+  }
+  ```
+  There was an [open issue](https://github.com/lodash/lodash/issues/4904) in their repository but got deleted (reason unknown). There's [another reported issue](https://github.com/lodash/lodash/issues/4945) explaining the problem, however it hasn't been resolved yet.
```

Comment:
There was an [open issue](https://github.com/lodash/lodash/issues/4904) in the lodash repository but got deleted (reason unknown). There's [another reported issue](https://github.com/lodash/lodash/issues/4945) explaining the problem, however it hasn't been resolved yet.